examples/logical_interconnects.py

```diff
@@ -48,6 +48,11 @@
 
 oneview_client = OneViewClient(config)
 
+# Get by ID
+print("Get a logical interconnect by ID")
+logical_interconnect = oneview_client.logical_interconnects.get(logical_interconnect_id)
```

Comment:
In this case, the execution must be interrupted since the Interconnect ID must be set by the user. To run the example, a logical interconnect should already exists.